docs/source/conf.py

```diff
@@ -227,10 +222,10 @@
 #html_show_sourcelink = True
 
 # If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
-#html_show_sphinx = True
+html_show_sphinx = False
 
 # If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
-#html_show_copyright = True
+html_show_copyright = False
```

Comment:
Why not?

Reply:
Because we use our own footer (see changes in this PR and discussion in "general" channel)

Reply:
Got it.